tex/main/MarkovChain/closed_form_state_probs/main.tex

```diff
@@ -0,0 +1,662 @@
+        \xrightarrow{\hspace*{2cm}} \hspace{1cm} \mu^6
+    \end{flalign*}
+\end{multicols}
+
+\begin{equation*}
+    \tilde{\pi}_{(0,0)} = (\lambda^A)^2 \mu^4 + 2 \lambda^A \mu^5 + \lambda^A \lambda^o \mu^4 + \mu^6
+\end{equation*}
+
+
+\newpage
+\begin{figure}[h]
+    \centering
+    \input{MarkovChain/closed_form_state_probs/example_model_1122/main.tex}
+\end{figure}
+
+\begin{multicols}{4}
+    \begin{figure}[H]
+        \centering
+        \scalebox{0.6}{\input{MarkovChain/closed_form_state_probs/spanning_trees_1122/main_0.tex}}
+    \end{figure}
+    \vspace*{\fill}
+    \columnbreak
+    \vspace*{0cm}
+    \begin{equation*}
+        (\lambda^A)^2 \mu^4
+    \end{equation*}
+    \vspace*{\fill}
+    \columnbreak
+    \begin{figure}[H]
+        \centering
+        \scalebox{0.6}{\input{MarkovChain/closed_form_state_probs/spanning_trees_1122/main_1.tex}}
+    \end{figure}
+    \vspace*{\fill}
+    \columnbreak
+    \vspace*{0.3cm}
+    \begin{equation*}
+        \lambda^A \mu^5
+    \end{equation*}
+\end{multicols}
+
+\begin{multicols}{4}
+    \begin{figure}[H]
+        \centering
+        \scalebox{0.6}{\input{MarkovChain/closed_form_state_probs/spanning_trees_1122/main_2.tex}}
+    \end{figure}
+    \vspace*{\fill}
+    \columnbreak
+    \vspace*{0.3cm}
+    \begin{equation*}
+        \lambda^A \mu^5
+    \end{equation*}
+    \vspace*{\fill}
+    \columnbreak
+    \begin{figure}[H]
+        \centering
+        \scalebox{0.6}{\input{MarkovChain/closed_form_state_probs/spanning_trees_1122/main_3.tex}}
+    \end{figure}
+    \vspace*{\fill}
+    \columnbreak
+    \vspace*{0.3cm}
+    \begin{equation*}
+        \mu^6
+    \end{equation*}
+\end{multicols}
+
+
+\begin{equation*}
+    \tilde{\pi}_{(0,0)} = (\lambda^A)^2 \mu^4 + 2 \lambda^A \mu^5 + \mu^6
+\end{equation*}
+
+\newpage
+\subsubsection{Conjecture of adding rows}
+
+Let us consider three Markov models with the same number of servers \(C=1\), the same threshold \(T=1\), the same system capacity \(N=2\) but different parking capacity \(M=1\), \(M=2\) and \(M=3\).
+
+
+\begin{multicols}{3}
+    \begin{figure}[H]
+        \centering
+        \scalebox{0.8}{\input{MarkovChain/closed_form_state_probs/example_model_1121/main.tex}}
+        \caption{\(M=1\)}
+    \end{figure}
+    \columnbreak
+    \begin{figure}[H]
+        \centering
+        \scalebox{0.8}{\input{MarkovChain/closed_form_state_probs/example_model_1122/main.tex}}
+        \caption{\(M=2\)}
+    \end{figure}
+    \begin{figure}[H]
+        \centering
+        \scalebox{0.8}{\input{MarkovChain/closed_form_state_probs/example_model_1123/main.tex}}
+        \caption{\(M=3\)}
+    \end{figure}
+\end{multicols}
+
+By increasing the parking capacity of the model it can be easily observed that the number of spanning trees rooted at \((0,0)\) increases as well since more combinations of paths can be generated using the new edges and vertices. The corresponding values of \(\tilde{\pi}_{(0,0)}\) of the three models are:
+
```

Comment:
```suggestion
By increasing the parking capacity of the system it can be observed that \(|T_{(0,0)}(G)|\) increases as well since 
more combinations of paths can be generated using the new edges and vertices. 
The corresponding values of \(\tilde{\pi}_{(0,0)}\) of the three systems are:

```